### DISCUSSION TITLE
Inserting footnotes and renumbering subsequent footnotes

### DISCUSSION THREAD

**Discussion by zsofiaj** [thegetty/quire#726] [Get Help] [answered]
If I need to add a footnote to an existing entry or essay, do I need to renumber any subsequent notes manually? Does anyone have any tips or tricks for getting these to renumber more efficiently? 

On the YAML and Markdown page of the Quire documentation, there is a note that says, "Warning! The built-in Markdown processor will automatically renumber footnotes in the order they appear in the text. It will also always put the footnotes at the very end of your content, no matter where you may try to put them." But I wasn't sure what this meant. 

I tried adding one note in the middle of an entry, and I couldn't figure out a great way to renumber the notes other than by manually changing the numbers both within the essay and in the notes at the bottom. 

I'm in Quire v1, 1.0.0-rc.3. 

Thank you!

**Comment by Erin-Cecele** [comment, marked as answer]
Hi @zsofiaj I recently had to do this for a Getty Quire publication. You will not only need to manually renumber the footnotes as they appear within the Markdown text, but you will also need to renumber the list of footnotes that appear at the bottom of the Markdown page. Let me know if this helps clear things up for you. 

<img width="287" alt="Screen Shot 2023-05-10 at 8 30 26 AM" src="https://github.com/thegetty/quire/assets/62075223/2620a68e-966e-443c-afe6-c9ff32847f9a">

<img width="300" alt="Screen Shot 2023-05-10 at 8 30 38 AM" src="https://github.com/thegetty/quire/assets/62075223/b1b38335-e3e3-40e7-816b-c5c38929df13">

**Reply by zsofiaj** [reply]
Thanks so much for confirming @Erin-Cecele! I did this for one entry just now (inserted a new note 12 in an entry with 17 notes), and it really only took a few minutes to renumber. So it is not such a bit deal to do it manually. I was just confused by that one note in the documentation and wondered if it was related.
